### PR TITLE
Keep scroll postion.

### PR DESCRIPTION
**Pull request by ivarconr**
/cc @sveisvei

### CODE REVIEW

File: src/component/app.jsx [hunk 1]
```diff
@@ -42,10 +43,6 @@ export default class App extends Component {
         if (this.props.location.pathname !== nextProps.location.pathname) {
             clearTimeout(this.timer);
             this.timer = setTimeout(() => {
-                window.requestAnimationFrame(() => {
-                    document.querySelector('.mdl-layout__content').scrollTop = 0;
```
Comment (ivarconr) [inline src/component/app.jsx]: @sveisvei Why did you add this in the first place?

File: src/component/app.jsx [hunk 2]
```diff
@@ -122,6 +119,7 @@ export default class App extends Component {
                             {createListItem('/applications', 'Applications', 'apps')}
                         </Navigation>
                     </Drawer>
+                    <ScrollContainer scrollKey="main">
```
Comment (ivarconr) [inline src/component/app.jsx]: seems to work with same key for everything. Might want to use the current path (window.location.pathname)?

File: src/component/app.jsx (hunk 1 above)
Comment (sveisvei) [inline src/component/app.jsx]: Because navigating a new page should send you to top.

File: src/component/app.jsx (hunk 2 above)
Comment (sveisvei) [inline src/component/app.jsx]: makes sense to lock to the pathname.

**Comment by sveisvei** [comment]
wfm (when using pathname instead of "main")

**Comment by ivarconr** [comment]
Kind of works. But feels strange to keep scroll position when using menu and not "back button".. @SimenB ?

**Comment by SimenB** [comment]
Can you check the navigation event? If it's go back (`POP`?) don't scroll?

**Comment by sveisvei** [comment]
Agree, it does not work out great when not using back. In general we just want it in list views.

**Comment by sveisvei** [comment]
btw, until we have tests we should just run the build - would have caught the missing dependency.

**Comment by ivarconr** [comment]
i can add that

**Comment by ivarconr** [comment]
scrollTo uses pathname under the hood to use correct scrolling per path. I don't see any reason to not have the same scrollKey in the ScrollContainer, it only makes sense if we have multiple scrollable containers on the same pathname.

**Comment by ivarconr** [comment]
It seems to be working now with a custom `shouldUpdateScroll`. It only scroll down when it sees a "POP" action.

**Comment by SimenB** [comment]
Nice! 🎉

**Comment by sveisvei** [comment]
wfm

**Comment by sveisvei** [comment]
💯
